bin/run-tests.sh

```diff
@@ -12,24 +12,22 @@
 # ./bin/run-tests.sh
 
 exit_code=0
+echo "current directory ${PWD}"
 
 # Iterate over all test directories
 for test_dir in tests/*; do
     test_dir_name=$(basename "${test_dir}")
     test_dir_path=$(realpath "${test_dir}")
-
-    bin/run.sh "${test_dir_name}" "${test_dir_path}" "${test_dir_path}"
-
-    # OPTIONAL: Normalize the results file
-    # If the results.json file contains information that changes between 
-    # different test runs (e.g. timing information or paths), you should normalize
-    # the results file to allow the diff comparison below to work as expected
+    test_out_path="/tmp/${test_dir_name}"
+    # Let Dyalog create the files to avoid its stdout (⎕←) length limit
+    bin/run.sh "runner-tests/$test_dir_name" "${test_dir_path}" "${test_out_path}"
```

Comment:
Why is the slug passed to `run.sh` equal to `"runner-tests/$test_dir_name"`?

Reply:
Right, the scripts do keep the positional argument for the slug, they just don't do anything with it right now. If a test needed the runner to adapt in some way that extension could be added in the future.

So, should the slug passed to `run.sh` be empty `""` or `"$test_dir_name"?` or something else?

Reply:
It should be `"$test_dir_name"`